src/react/databrowser/objects/ObjectRow.jsx

```diff
@@ -58,9 +61,17 @@ const Row = ({
   const row = rows[index];
   prepareRow(row);
 
+  const { pathname } = useLocation();
+  const query = useQuery();
+
   const handleClick = () => {
     dispatch(toggleAllObjects(false));
     dispatch(toggleObject(row.original.key, row.original.versionId));
```

Comment:
I think this line may be dropped as the ObjectListTable will dispatch it when the url change